tests/data/lint.cgx

```diff
@@ -74,8 +74,8 @@ from PySide6.QtWidgets import QBoxLayout
 
 
 class Directives(cg.Component):
-    def __init__(self, props):
-        super().__init__(props)
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Should it be init hook?

Reply:
I haven't released collagraph with this hook yet :') but makes sense